docbook/reference/en/en-US/modules/Installation_Configuration.xml

```diff
@@ -248,13 +248,44 @@
 	        <title>Upgrading RESTEasy within WildFly</title>
 	        <para>
 	            RESTEasy is bundled with WildFly, but you may want to upgrade RESTEasy in WildFly to the latest version.  The RESTEasy
-	            distribution comes with a zip file called resteasy-jboss-modules-&lt;version&gt;.zip.  Unzip this file
+                distribution comes with a zip file called <code><![CDATA[resteasy-jboss-modules-<version>.zip]]></code>.  Unzip this file
 	            within the modules/system/layers/base/ directory of the WildFly distribution.  This will configure WildFly
 	            to use new versions of the modules listed in <xref linkend="resteasy_modules_in_wildfly"/>.
 	        </para>
+            <para>
+                To build the <code><![CDATA[resteasy-jboss-modules-<version>.zip]]></code>, enter the <code>jboss-modules</code>
+                directory of the project, and run the following Maven command:
+                <programlisting><![CDATA[$ mvn install]]></programlisting>.

```

Comment:
The "." after </programlisting> comes out at the beginning of the line after the code. Make it go away!! ;-)

Reply:
Okay!